public/pages/CreateTrigger/utils/constants.js

```diff
@@ -32,6 +32,11 @@ export const FORMIK_INITIAL_ACTION_VALUES = {
     lang: 'mustache',
     source: DEFAULT_MESSAGE_SOURCE,
   },
+  throttle_enabled: true,
```

Comment:
np: Do we want to enable this by default for all the new actions?

Reply:
I agree, I'd rather us by noisy by default where people can then choose to throttle rather than have users potentially miss alerts because they didn't fully understand they are being throttled.

Reply:
Offline talk with Eli, will change the default value to False. Thanks